pkgs/development/python-modules/bx-python/default.nix

```diff
@@ -2,38 +2,51 @@
   lib,
   fetchFromGitHub,
   buildPythonPackage,
-  pythonOlder,
   numpy,
   cython,
   zlib,
   python-lzo,
-  nose,
+  pytestCheckHook,
+  setuptools,
+  oldest-supported-numpy,
 }:
 
 buildPythonPackage rec {
   pname = "bx-python";
-  version = "0.11.0";
-  format = "setuptools";
-
-  disabled = pythonOlder "3.8";
+  version = "0.12.0";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "bxlab";
     repo = "bx-python";
     rev = "refs/tags/v${version}";
-    hash = "sha256-evhxh/cCZFSK6EgMu7fC9/ZrPd2S1fZz89ItGYrHQck=";
+    hash = "sha256-ZpZjh7OXdUY7rd692h7VYHzC3qCrDKFme6r+wuG7GP4=";
   };
 
-  nativeBuildInputs = [ cython ];
+  postPatch = ''
+    substituteInPlace pytest.ini \
+      --replace-fail "--doctest-cython" ""
+  '';
+
+  build-system = [
+    setuptools
+    cython
+  ];
 
   buildInputs = [ zlib ];
 
   propagatedBuildInputs = [
     numpy
+    oldest-supported-numpy
```

Comment:
Apparently this should go in `build-system` (cf. https://pypi.org/project/oldest-supported-numpy/).

Reply:
Moved it